community/config/eggs/creating_a_custom_image.md

```diff
@@ -142,18 +176,18 @@ curly braces `{{EXAMPLE}}` with a matching environment variable (such as `EXAMPL
 :::
 
 ```bash
-java -Xms128M -Xmx{{SERVER_MEMORY}}M -jar {{SERVER_JARFILE}}
+java -Xms128M -XX:MaxRAMPercentage=95.0 -jar {{SERVER_JARFILE}}
 ```
 
 Becomes:
 
 ```bash
-java -Xms128M -Xmx1024M -jar server.jar
+java -Xms128M -XX:MaxRAMPercentage=95.0 -jar {{SERVER_JARFILE}} server.jar
```

Comment:
Here you should be left with just `server.jar` without the `{SERVER_JARFILE}}`.

Reply:
thank you!

Reply:
@MinerPL You know what is wrong with this change that the build is faling?

Reply:
@QuintenQVD0 Without this line 
```md
::: tip
If you want to use egg non stock variables in the configuration parser you must reference them as `{{server.build.env.ENVNAME}}` or just `{{env.ENVNAME}}`. Do not forget to to replace `ENVNAME` with the actual enviroment name you have setup.
:::
``` Everything build correct so probably here is a mistake that fail builds.

Reply:
@QuintenQVD0 Without this line 
```md
::: tip
If you want to use egg non stock variables in the configuration parser you must reference them as `{{server.build.env.ENVNAME}}` or just `{{env.ENVNAME}}`. Do not forget to to replace `ENVNAME` with the actual enviroment name you have setup.
:::
``` Everything build correct so probably here is a mistake that fail builds.

Reply:
Okay, the problem is with `{{server.build.env.ENVNAME}}` and `{{env.ENVNAME}}`